### DISCUSSION TITLE
part-ia/2012-9/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#425] [Question comments]
Part IA 2012 Differential Equations: Paper 2, Section II, A 

https://questions.tripos.org/part-ia/2012-9/

**Comment by hennalVEVO** [comment]
<details>
<summary>Solution outline</summary>


Set $\nabla V = 0$ and solve for the stationary points. Determine type of stationary point by looking at the signature of each. Draw contours around the maxima/minima first, then saddle points. Use multivariate chain rule to show that $\frac{dV}{dt} < 0$. Use your sketch and the non-increasing nature of $V$ to justify the behaviour as $t \to \infty$.

</details>
